services/director-v2/tests/unit/with_dbs/test_api_route_computations.py

```diff
@@ -225,19 +229,30 @@ def mocked_catalog_service_fcts_deprecated(
     def _mocked_services_details(
         request, service_key: str, service_version: str
     ) -> httpx.Response:
+        data_published = fake_service_details.model_copy(
+            update={
+                "key": urllib.parse.unquote(service_key),
+                "version": service_version,
+                "deprecated": (
+                    dt.datetime.now(tz=dt.UTC) - dt.timedelta(days=1)
+                ).isoformat(),
+            }
+        ).model_dump(by_alias=True)
+
+        deprecated = {
+            "deprecated": (
+                dt.datetime.now(tz=dt.UTC) - dt.timedelta(days=1)
+            ).isoformat()
+        }
+
+        data = {**ServiceGet.model_config["json_schema_extra"]["examples"][0], **data_published, **deprecated}  # type: ignore
```

Comment:
the type ignore that you are putting everywhere when using json_schema_extra can be fixed by using assertions.
this has the advantage of not ignoring potential pitfalls.

Reply:
And 2nd question:
@pcrespov @giancarloromeo did you adjust the model of ServiceGet for the deprecation field? or where is it hidden now?

Reply:
The `deprecated` field was in the `ServiceGet` and the mock was wrong.